### DISCUSSION TITLE
Is it possible to use MYSQL instead of Mongodb?

### DISCUSSION THREAD

**Discussion by GWLdeveloperAjeetesh** [kinngh/shopify-node-express-mongodb-app#99] [Ideas]
if it is possible then how can i do that?

**Comment by kinngh** [comment]
Why wouldn't it be? Just replace the MongoDB with SQL and if you use an ORM, use that.

**Comment by GWLdeveloperAjeetesh** [comment]
@kinngh 
Could you have any suggestion, what would be best way to do that?

**Comment by kinngh** [comment]
@GWLdeveloperAjeetesh Swap out all instances of MongoDB models with MySQL models, add in your connection string, look into the code where all there's a model import and add in your queries / ORM and you're good to go. If you want to see a reference, take a look at the [Next.js x Prisma ORM](https://github.com/kinngh/shopify-nextjs-prisma-app) repo by searching for all instances of `prisma` and swapping it out here with your MySQL stack.

I'm under the assumption you know what you're doing, have worked with MySQL before and know what you're doing when it comes to Express and React.

**Comment by GWLdeveloperAjeetesh** [comment]
Understood, thanks alot @kinngh
